eclass/rpm-extended.eclass

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2020 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
For cross-reference, the python-single-r1.eclass is a good place to look at for my comments: https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/python-single-r1.eclass